Automattic/wp-calypso
# Replace Yoast banner with Blaze banner on /stats/ page

#### Proposed Changes

* Replace the Yoast SEO banner on /stats with a banner for Blaze.

Before | After
--|--
![stats-banner-before](https://user-images.githubusercontent.com/140841/208198615-4642fe16-316b-4780-a8bb-d945eac333b0.png)  | ![stats-banner-after](https://user-images.githubusercontent.com/140841/208198629-5b4b5207-e86d-4a37-91e8-75a324707906.png)

#### Testing Instructions



* Load this PR.
* Ensure clicking the banner CTA takes you to `https://wordpress.com/advertising/{blog-domain}`
* Ensure Tracks are being recorded for impressions and clicks
* Make sure banner appears on Simple, Atomic, and Jetpack connected sites
* Ensure the banner matches the Figma. pb5gDS-2fR-p2
* Ensure the secondary Jetpack mobile app banner still functions correctly.

#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/issues/70835
